src/app-layout/__tests__/toolbar.test.tsx

```diff
@@ -34,12 +34,6 @@ describe('toolbar mode only features', () => {
       expect(findToolbar(wrapper)).toContainElement(wrapper.findToolsToggle().getElement());
     });
 
-    test('does not render navigation toggle button for open state', () => {
-      const { wrapper } = renderComponent(<AppLayout navigationOpen={true} onNavigationChange={noop} />);
-      expect(findToolbar(wrapper)).toBeTruthy();
-      expect(wrapper.findNavigationToggle()).toBeFalsy();
```

Comment:
We should keep the test, but update this assertion to the new behavior, `.toBeTruthy()`